jina/docker/hubio.py

```diff
@@ -258,8 +265,6 @@ def _docker_login(self) -> None:
                 self.logger.debug(f'successfully logged in to docker hub')
             except APIError:
                 raise DockerLoginFailed(f'invalid credentials passed. docker login failed')
-        else:
-            raise DockerLoginFailed('no username/password specified, docker login failed')
```

Comment:
At least log something in the case of failure?

Reply:
no credentials will follow docker host auth context, therefore this `raise` is unnecessary in the first place